hooks/chat/useBots.tsx

```diff
@@ -11,13 +11,15 @@ const generateBot = () => faker.internet.userName()
 
 const BOTS_ENABLED = process.env.NEXT_PUBLIC_WITH_BOTS === "true"
 // How often the bot will send a message in milliseconds
-const BOT_INTERVAL = parseInt(process.env.NEXT_PUBLIC_BOT_INTERVAL || "60000")
-// How many bots to spawn
-const BOT_COUNT = parseInt(process.env.NEXT_PUBLIC_BOT_COUNT || "100")
-// How many bots will publish messages
-const BOT_PUBLISHER_PROBABILITY = parseFloat(
-  process.env.NEXT_PUBLIC_BOT_PUBLISHER_PROBABILITY || "1"
-)
+const rawInterval = parseInt(process.env.NEXT_PUBLIC_BOT_INTERVAL ?? "60000", 10);
+const BOT_INTERVAL = Number.isNaN(rawInterval) || rawInterval <= 0 ? 60000 : rawInterval;
```

Comment:
```suggestion
const BOT_INTERVAL = Number.isNaN(rawInterval) || rawInterval < 10000 ? 60000 : rawInterval;
```
To keep the `BOT_INTERVAL-10000` expression from useBots positive.